static/index.html

```diff
@@ -29,6 +29,31 @@ <h1><img src="static/images/c4sj_logo.jpg" alt="Code For San Jose" id="c4sj_logo
         <span>Email address</span>
         <input type="email" name="email" placeholder="your email address" required>
       </label>
+      <br/>
+      <legend>Preferred Pronoun:</legend>
+      <form action="" method="post">
```

Comment:
HTML form elements should not be nested. Also, this would work better as a radio button, since you can't check more than one option if it's a `type="radio"`, and it doesn't make sense in this instance to be able to select more than one option.

For the show/hide behavior, we can have another input field that's hidden by default, then becomes visible via javascript  if the `other` radio button is selected.  On the form handling side (the Python code), this would need to also ignore the `other` text field in most cases, but then use the text that's provided only if the `other` radio button is selected.